caiman/source_extraction/cnmf/online_cnmf.py

```diff
@@ -80,6 +81,26 @@ class OnACID(object):
     """
 
     def __init__(self, params=None, estimates=None, path=None, dview=None, Ain=None):
+        """
+
+        Args:
+            params: CNMFParams
+                CNMFParams object with parameters for the entire run of initialization and OnACID
+
+            estimates: Estimates, optional
+                Estimates object to load an existing model
+
+            path: str, optional
+                path to a saved OnACID model on disk
+
+            dview:
+                dview instance
+
+            Ain: csc_matrix, optional
```

Comment:
I couldn't find if there's a place where it gets converted to a regular dense array, if a dense array is provided. If you can check and test that it works with dense it could be modified.

Reply:
Regardless, I don't like `csc_matrix`, many people don't know what that means maybe change to `sparse (csc) matrix` so newer people will understand it? I think we should probably do this all over the docs.

Reply:
Ths is the type, and in docstrings the type annotation should be presented as-is. There's a description on the next line which explains it more :)

Reply:
Yes you are right.